ui/imports/shared/panels/ModuleWarning.qml

```diff
@@ -31,8 +32,12 @@ Item {
     signal hideFinished()
 
     function show() {
+        if (localAppSettings.testEnvironment) {
```

Comment:
I'm not sure if this is the right place to do this.
Since this is just a component, I'd say that it shouldn't check some global flags like this one.

I'd prefer that we apple `localAppSettings.testEnvironment` to set `active` property in `AppMain.qml`.

That would also give more space if we decide to hide only some banners, not all of them.

Reply:
I get your point. I did wonder the same. I think we never want to ever show any banner in tests though, because just having one can break the test.

I also don't particularly like for a specific component to access the settings, but I wanted to make it future proof where if we add a new banner, we don't have to remember to apply the condition.

We can wait and see what others think.

Reply:
I agree, thought the same before writing 
Let's leave it as is, any way it's just already done 🙂

Reply:
Well, we could, in some point, want to test the banners' functionality itself.. Then we should need to modify this code. For now we can keep it like this but.. maybe we can add a comment saying it is a temporal solution?!